serverless-chatbot/handler.js

```diff
@@ -0,0 +1,35 @@
+const { SDK } = require('beekeeper');
+const { MessageTypes, toMessage } = require('beekeeper/dist/model/message');
+
+const credentials = {
+    tenantURL: process.env.BEEKEEPER_TENANT_URL,
+    token: process.env.BEEKEEPER_API_TOKEN,
+};
+
+const sdk = new SDK(credentials);
+
+const messageReceived = event => {
+    const notification = JSON.parse(event.body);
+    const { payload: { message: { conversation_id, text } } } = notification;
```

Comment:
You have to handle also other types of messages - not everything will have the text (like images, voice messages etc.).
You could just filter them out or handle in some other way.

Reply:
Good point.